debian/control

```diff
@@ -8,7 +8,7 @@ Vcs-Git: https://github.com/endlessm/eos-phone-home.git
 
 Package: eos-phone-home
 Architecture: all
-Depends: wget, python, ${misc:Depends}
+Depends: wget, python3, python3-requests, ${misc:Depends}
```

Comment:
I used urllib intentionally vs requests since it's a base python lib as I'm a bit wary of external libs in this type of code but if that's not a concern it indeed does collapse the code a bunch.